zerver/actions/message_edit.py

```diff
@@ -600,21 +603,19 @@ def update_message_content(
     )
 
 
-def apply_automatic_unmute_follow_topics_policy(
+def automatic_unmute_follow_topics_policy(
     user_profile: UserProfile,
     target_stream: Stream,
     target_topic_name: str,
-) -> None:
+) -> tuple[bool, bool]:
```

Comment:
I think it would be cleaner to split this function into two smaller functions: `should_follow_target_topic` and `should_unmute_target_topic`.

I also noticed that the `target_topic_name` parameter is not being used anymore.